pandas/tests/series/test_operators.py

```diff
@@ -1433,6 +1433,20 @@ def test_ops_datetimelike_align(self):
         result = (dt2.to_frame() - dt.to_frame())[0]
         assert_series_equal(result, expected)
 
+    def test_bool_ops_with_index(self):
+        # GH#19792
+        # TODO: reversed ops still raises, GH#22092
```

Comment:
Can we have an `xfail` test for that in this PR?

(unless you plan to patch the reversed op in the near term, then not needed)

Reply:
+1 on this idea.  I'll wait til AM to implement it, ideally on the follow-up to #22153.

Reply:
can you add this test?